api/res/openapi.yml

```diff
@@ -185,6 +221,36 @@ paths:
             text/plain:
               schema:
                 type: string
+  /tasks:
+    get:
+      responses:
+        '200':
+          description: The list of pending and active tasks
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/TaskCompletionState'
+  /tasks/{taskId}:
+    parameters:
+      - name: taskId
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+    get:
+      summary: Get the state of the given task
+      description: >-
+
+      responses:
+        '200':
+          description: The state of the task
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TaskCompletionState'
```

Comment:
For the time being, I removed the task response altogether.

I still would like to see it working as initially intended, so that you can get the original response after polling, maybe even after the task completed, but one step at a time, right? :)

So, I described it as it's currently implemented:
- As long as the task endpoint does not return 404, the task is running.
- Once you get a 404, the task is finished.
- The polling resource does not have a response body.